### ISSUE TITLE
Images are not shown in CONTRIBUTING.md

### ISSUE THREAD

**Issue by prajjwalarora** [covid19india/covid19india.github.io#2358]
###  Description 
This issue in the process flow in the CONTRIBUTING.md guide. Images added in this section are not visible(the only Alt text is visible) and I found that the link in src of these images is not working(closed).

**To Reproduce**

1. Go to 'https://github.com/prajjwalarora/covid19india-react/blob/master/CONTRIBUTING.md'
2. Scroll down to 'process flow section'

**Expected behavior**
The image Should visible and if there is no image available for this then img tag removed and the text should be formated.

**Screenshots**
![Annotation 2020-10-01 180847](https://user-images.githubusercontent.com/57281080/94811149-9d4e2a00-0412-11eb-9046-2193e68c9b5e.png)

**Desktop (please complete the following information):**

- OS: Windows 10 20H1
- Browser: Chrome, edge
- Version: 85,85

**Comment by PPInfy** [comment]
I think this can be closed. It is just few lines change. Someone can merge the contributing.md only.  @prajjwalarora

**Comment by prajjwalarora** [comment]
@PPInfy yes this can be done by this but problem is that merge request cannot be made without issue has been assigned as per there terms.

**Comment by shuklaayush** [comment]
Fixed via 059fc13fbc9d13495971926a4c41e3abdb0140ce
